internal/boxcli/global.go

```diff
@@ -98,6 +100,16 @@ func globalPullCmd() *cobra.Command {
 	}
 }
 
+func globalShellenvCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "shellenv",
+		Short: "Print shell commands that add Devbox packages to your PATH",
+		Run: func(*cobra.Command, []string) {
+			fmt.Print(impl.GenerateShellEnv())
+		},
+	}
```

Comment:
@loreto had suggested using `devbox activate` for this. I like that a bit more because then it's not related to `global` but actually a more general command that let's us modify and improve rc configuration over time. Thoughts?